storage-operator/pkg/apis/storage/v1alpha1/volume_types.go

```diff
@@ -63,44 +63,72 @@ type PersistentVolumeTemplateSpec struct {
 
 type VolumePhase string
 
-// "Enum" representing the phase/status of a volume.
+// TODO: kept for temporary compatibility, to be removed.
+// "Enum" representing the phase of a volume.
```

Comment:
So we still have a phase for a volume but it's not directly in the CRD but it comes from the computation of all conditions?

Reply:
Oh I've just discovered the `computePhase` later, so I guess that's it.

Reply:
Yeah, we still have the concept of a global "Phase" because lot of logic relies on that.

Not sure we will keep it when we make the real move to conditions, we will see.
But for now we have a computed one to ease the transition.